<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorWindow</class>
 <widget class="QMainWindow" name="AdministratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администратор</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/settings.png</normaloff>../icons/settings.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabBar::tab { min-width: 100px; min-width: 100px; }</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>../icons/employee.png</normaloff>../icons/employee.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QTableWidget" name="tableEmployee">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>ФИО</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Логин</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Роль</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Статус</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="addEmployee">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Добавить сотрудника</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset>
         <normaloff>../icons/open-book.png</normaloff>../icons/open-book.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QPushButton" name="addAuthor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTipDuration">
           <number>-3</number>
          </property>
          <property name="text">
           <string>Добавить автора</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="addGenre">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Добавить жанр</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox1">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Издательства</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tablePublisher">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="addPublisher">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Добавить издателя</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QGroupBox" name="groupBox2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Авторы</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableAuthor">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QGroupBox" name="groupBox3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Жанры</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableGenre">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCommandLinkButton" name="logout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/exit.png</normaloff>../icons/exit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addEmployee</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>goToAddEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tablePublisher</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>addPublisher(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPublisher</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>addRowPublisher()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addGenre</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>addRowGenre()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>676</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addAuthor</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>addRowAuthor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableAuthor</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>addAuthor(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableGenre</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>addGenre(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>676</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableEmployee</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>goToChangeEmployee(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logout</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorWindow</receiver>
   <slot>logout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goToAddEmployee()</slot>
  <slot>addRowPublisher()</slot>
  <slot>addRowAuthor()</slot>
  <slot>addRowGenre()</slot>
  <slot>addPublisher(QTableWidgetItem*)</slot>
  <slot>addAuthor(QTableWidgetItem*)</slot>
  <slot>addGenre(QTableWidgetItem*)</slot>
  <slot>goToChangeEmployee(QTableWidgetItem*)</slot>
  <slot>logout()</slot>
 </slots>
</ui>
